about.html

```diff
@@ -7,6 +7,7 @@
     <meta name="author" content="Mikael Elias">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Mikael Elias</title>
```

Comment:
To pensando agora, talvez seja bom colocar seu nome como prefixo no titulo de todas as páginas tipo assim:
```suggestion
    <title>Mikael Elias - About</title>
```

Reply:
Feito!